XCUITests/AuthenticationTest.swift

```diff
@@ -17,13 +17,19 @@ class AuthenticationTest: BaseTestCase {
     }
 
     func testTurnOnOff() {
+        waitForExistence(app.textFields["url"], timeout: 5)
```

Comment:
not sure if we have to wait for app.textFields["url"] or better for app.buttons["urlBar-cancel"] ... I guess that if you got tests running, then that's fine too

Reply:
While testing locally, I had intermittent failures. Adding that line made the tests pass for me.